src/main/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilder.java

```diff
@@ -0,0 +1,76 @@
+package bio.terra.service.snapshotbuilder.utils;
+
+import bio.terra.service.snapshotbuilder.query.FieldPointer;
+import bio.terra.service.snapshotbuilder.query.FieldVariable;
+import bio.terra.service.snapshotbuilder.query.FilterVariable;
+import bio.terra.service.snapshotbuilder.query.Literal;
+import bio.terra.service.snapshotbuilder.query.Query;
+import bio.terra.service.snapshotbuilder.query.TableNameGenerator;
+import bio.terra.service.snapshotbuilder.query.TablePointer;
+import bio.terra.service.snapshotbuilder.query.TableVariable;
+import bio.terra.service.snapshotbuilder.query.filtervariable.BinaryFilterVariable;
+import bio.terra.service.snapshotbuilder.query.filtervariable.BooleanAndOrFilterVariable;
+import bio.terra.service.snapshotbuilder.query.filtervariable.FunctionFilterVariable;
+import java.util.List;
+
+public class SearchConceptsQueryBuilder {
+
+  private SearchConceptsQueryBuilder() {}
+
+  public static String buildSearchConceptsQuery(
+      String domainId, String searchText, TableNameGenerator tableNameGenerator) {
+    var conceptTablePointer = TablePointer.fromTableName("concept", tableNameGenerator);
+    var conceptTableVariable = TableVariable.forPrimary(conceptTablePointer);
+    var nameField = conceptTableVariable.makeFieldVariable("concept_name");
+    var idField = conceptTableVariable.makeFieldVariable("concept_id");
```

Comment:
I honestly hate to bring this up, but given the query only references the concept table, we could relatively easily add this new endpoint to the existing azure integration and gcp connected tests. It might be worth a larger team discussion to determine our testing strategies for changes like this.

Reply:
I'm definitely interested in talking about this more!

Reply:
Thanks for chatting about this in standup - I think adding this with the limit clause work makes sense.